doc/source/getting-started/quickstart.rst

```diff
@@ -23,6 +23,12 @@ Instead, if you use `conda <https://anaconda.org/>`_ type:
 
 Here an example of Quantum Fourier Transform (QFT) to test your installation:
 
+.. testsetup::
+
+   from qibo.backends import set_backend
+
+   set_backend("numpy")
+
```

Comment:
Why is this being inserted here?

Reply:
It's just a little setup in order to be sure that the numpy backend and not the tensorflow one is used in the following testcode.

Reply:
But it is setting up tests that are unrelated to the Clifford backend, if I understood correctly

Reply:
Ah, yeah, I think I was getting an error because it was using the clifford backend with tensorflow as engine. I believe that happened since the last test of the api reference is the clifford backend one. Therefore, I decided to force the usage of the numpy backend just to avoid possible problems.

Reply:
It is forcing the usage in all tests. If this is necessary, it should be forced only where the Clifford backend tests are